docs/jazzy.yml

```diff
@@ -7,8 +7,8 @@ copyright: '© 2014–2017 [Mapbox](https://www.mapbox.com/). See [license](http
 
 head: |
   <link rel='shortcut icon' href='https://www.mapbox.com/img/favicon.ico' type='image/x-icon' />
-#skip_undocumented: Yes
-#hide_documentation_coverage: Yes
+skip_undocumented: Yes
+hide_documentation_coverage: Yes
```

Comment:
As an alternative to hiding coverage information altogether, if there are any symbols we intentionally avoid documenting, jazzy skips over any symbol with a `:nodoc:` comment.